packages/vector_graphics_compiler/lib/vector_graphics_compiler.dart

```diff
@@ -116,12 +116,20 @@ Future<Uint8List> encodeSVG(String input, String filename) async {
         break;
       case DrawCommandType.vertices:
         final IndexedVertices vertices =
-            instructions.vertices[command.objectId];
+            instructions.vertices[command.objectId as int];
         final int fillId = fillIds[command.paintId]!;
         assert(!strokeIds.containsKey(command.paintId));
         codec.writeDrawVertices(
             buffer, vertices.vertices, vertices.indices, fillId);
         break;
+      case DrawCommandType.opacity:
+        codec.writeOpacityLayer(buffer, command.objectId as double);
```

Comment:
This will need to encode a paint

Reply:
https://github.com/dnfield/vector_graphics/pull/10#discussion_r823105486

Reply:
Done